### DISCUSSION TITLE
If CI will cache .rugby

### DISCUSSION THREAD

**Discussion by ghost** [swiftyfinch/Rugby#95] [Q&A] [answered]
Hi @swiftyfinch 

![image](https://user-images.githubusercontent.com/93128097/143666453-982bea17-7692-48ee-9134-e30630f477c9.png)

Does any unnecessary folder in build folder, mean that I don't need CI cache it?

And I tried to add the following into `.gitignore` but not working (plans.yml is ignored.), or would you mind provide CMD like this `mint run rugby plans --path --plan` as alternatively way?

```
# .gitignore 
.rubgy
!.rugby/plans.yml
```

**Comment by swiftyfinch** [comment, marked as answer]
Hey @Wei-Pinkoi, I hadn't tried to use 🏈 Rugby cache on CI yet.
I think you can ignore these files:
```sh
# Keep .rugby/cache.yml and .rugby/build/*-iphone*
.rugby/*
!.rugby/build
.rugby/build/*
!.rugby/build/*-iphone*
!.rugby/cache.yml
```
And use key as `hash('Podfile.lock')`. It depends on your CI tools.

I'm not sure about the custom `.rugby/plans.yml` path. Are you sure that you need it?
Please, describe the case more verbose. You can open a new feature request for this.

**Reply by ghost** [reply]
Oh now .gitignore working, I must tired last few days.

About custom path, or idea is that I think those config can put under the project root path, example:
and CI can easy cache dependancies under `.rugby/build/` (only have \*-iphone\*) with Podfile.lock (not much ignore rules.)
```
Project.xcworkspace
Pods/
Gemfile
Gemfile.lock
Podfile
Podfile.lock
Rugbyplan (as plans.yml)
Rugbyplan.lock (as cache.yml)
.rugby/build/
```

**Reply by swiftyfinch** [reply]
Yeah, it can be useful to keep common `plans.yml` under git and also have one private plans file per user.

**Reply by swiftyfinch** [reply]
@Wei-Pinkoi I've added a new feature request #97

**Reply by ghost** [reply]
Thanks.
